modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_uploader/js/components/DragFileIcon.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import React from 'react';
+
+export default function dragFileIcon() {
+	return (
+		<svg
```

Comment:
I was wondering if this icon should be in the Clay icon library.

Reply:
Sorry, it´s a compound icon/illustration not only a simple icon but we have an embedded one here. 

We planned to update icons in the future via prop, but we need to know the type of files in advance to implement it. We will do it on a separate ticket.

Reply:
Adding more context, the icon part is small

![image](https://user-images.githubusercontent.com/67901/149136988-e26ab402-4b32-4bd1-83e8-1353246ad76e.png)

and we planned to make it configurable via prop but we don't have the `type` right now in front.

We plan to add it later in a separate task. Do you think it's OK?

Reply:
:+1: Sure @boton I was mostly raising some doubts, but if you're going to keep on working on this in the future then it's fine by me